cluster-scope/base/core/namespaces/tremor-demo/kustomization.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+    - namespace.yaml
+components:
+    - ../../../../components/project-admin-rolebindings/tremor-demo
+    - ../../../../components/limitranges/default
```

Comment:
@GaryPWhite can you please add a resourcequota here? 
You can see an [example kustomization](https://github.com/operate-first/apps/blob/master/cluster-scope/base/core/namespaces/ds-github-labeler/kustomization.yaml#L8). 
Also, [here](https://github.com/operate-first/apps/tree/master/cluster-scope/components/resourcequotas) is the list of available resource quotas.
If you don't like any of the existing resource quotas, you can create a custom one ([example](https://github.com/operate-first/apps/tree/master/cluster-scope/base/core/namespaces/opf-trino-stage))

Reply:
can do!